crypto/err/err.c

```diff
@@ -80,6 +80,7 @@ static ERR_STRING_DATA ERR_str_libraries[] = {
     {0, NULL},
 };
 
+/* NOTE: MUST put here all reasons from include/openssl/err.h.in */
```

Comment:
May I suggest something like this?

``` C
/* For maintainability, please keep all reasons in the same order as in include/openssl/err.h.in */
```

(since there's no MUST, technically speaking)

Reply:
Well, at least all the reasons SHOULD be present. Changed to:
```
/*
 * Should make sure that all ERR_R_ reasons defined in include/openssl/err.h.in
 * are listed.  For maintainability, please keep all reasons in the same order.
 */
```